### DISCUSSION TITLE
Screenshots

### DISCUSSION THREAD

**Discussion by Visone-Selektah** [nwg-piotr/nwg-shell#215] [Ideas]
Hi!

A module or applet to manage the different types of screenshots available, focus window, area, output, etc.

**Comment by nwg-piotr** [comment]
What exactly do you mean? We already have a [script](https://github.com/nwg-piotr/nwg-shell/blob/main/scripts/screenshot) that does all the possible kinds of screenshots.

**Reply by Visone-Selektah** [reply]
Yes, I know, I was talking about some graphical applet | module to run that script, some like the check updates, that you click and run some command/script

**Reply by nwg-piotr** [reply]
Well, I'm not sure if there's a huge demand on it. Users of sway/Hyprland mostly use key bindings, as far as I know. Perhaps a survey on this matter would be useful.

**Reply by nwg-piotr** [reply]
After thinking about it, I found it a good idea. I added an "applet" in the system tray. Testing now.

![image](https://github.com/nwg-piotr/nwg-shell/assets/20579136/7b26ed60-b168-4e4c-9e17-f7a6e1f85d49)
